app/views/account/navigation_bottom.blade.php

```diff
@@ -1,6 +1,12 @@
 <div class="footer">
 	<div class="container">
-		<b class="copyright">&copy; 2015 - Automated Library Management System </b> All rights reserved.
+		<b class="copyright">&copy; <span id="ccdate">2015</span> - Automated Library Management System </b> All rights reserved.
 		<br><a href="https://github.com/prabhakar267/library-management-system" target="_blank">Fork on Github</a>
 	</div>
-</div>
\ No newline at end of file
+</div>
+
+<script type="text/javascript">
```

Comment:
Since this is PHP template, you don't need to add javascript for this
You can refer to #9